src/api/cubrid_log.c

```diff
@@ -1857,6 +1847,9 @@ cubrid_log_reset_globals (void)
 {
   int i;
 
+  css_free_conn (g_conn_entry);
```

Comment:
How about adding 'if (g_conn_entry != NULL)' check? (The g_conn_entry is needed to initialize NULL)